js/index.js

```diff
@@ -0,0 +1,34 @@
+//fetch github repos
+
+fetch("https://api.github.com/users/lemonadelightice/repos")
+ .then((response) => {
+    if(response.ok) {
+        return response.text();
+    } else {
+        throw new Error("Failed to fetch repositories");
+    }
+ })
+ .then((data) => {
+    const repositories = JSON.parse(data);
+    console.log(repositories);
+ 
+ //DOM to select Projects section by ID

```

Comment:
This code is fine but the comments are not needed. Other people reading this code most likely know what it means. Comments are really to add additional insight that one would not get from the actual code.